components/brave_welcome_ui/components/welcome/style.ts

```diff
@@ -6,25 +6,49 @@
 import styled from 'styled-components'
 
 export const Box = styled.div`
-  background: rgba(255, 255, 255, 0.1);
-  backdrop-filter: blur(15px);
-  border-radius: 30px;
+  position: relative;
   max-width: 800px;
+  border-radius: 30px;
   color: white;
   font-family: ${(p) => p.theme.fontFamily.heading};
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
   text-align: center;
-  position: relative;
+
+  .view-backdrop {
+    background: rgba(255, 255, 255, 0.1);
+    backdrop-filter: blur(15px);
+    border-radius: 30px;
+    position: absolute;
+    top: 0;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+
+    &.initial {
+      scale: 0.9;
```

Comment:
What does this achieve? I'm not seeing that there is a transition on this property. Perhaps a comment as to what the purpose of `.initial` is would help.

Reply:
When we animate specific styles of an element we define an initial state. This depends on the kind of animation of course. 

This initial state here shouldn't be part of the base styles because we perform a check if animations should play or not. You can see it being expressed below:

`<div className={classnames({ 'view-content': true, 'initial': shouldPlayAnimations })}>`

Reply:
Right but there isn't a transition on scale, so does this just jump from 0.9 to 1 when the initial class is removed?